packages/documentation/docs/vuefire/binding-subscriptions.md

```diff
@@ -4,14 +4,14 @@ In Vuefire, subscriptions to changes are handled transparently, that's why we al
 
 There are two ways of binding a Reference to the Database with Vuefire:
 
-- Using the `firebase`/`firestore` option
-- Calling the injected methods `$bind`/`$rtdbBind`
+- Simple "declarative" method, which you can only use when you are always binding to the same source, for example, a data structure accessible to all users. This uses the `firebase` or `firestore` option inside `export default`.
```

Comment:
I think it's better to keep it short here and add some details below in each section. I also think it would be better to match the titles, so keeping _Declarative binding_ would be nice.

Reply:
Good thinking